tooling/helpers/rest_helper.py

```diff
@@ -1,14 +1,19 @@
+from http import HTTPStatus
 import requests
 import time
 
+from helpers.data_helper import RestResult
 
-def await_transcription_finish(id: str, api_key: str):
-    is_finished = False
-    # TODO: add time cap
-    while not is_finished:
+# 30 minutes in seconds
+TRANSCRIPT_WAIT_MAX_TIME = 30 * 60
+TRANSCRIPT_ITERATION_WAIT_TIME = 10
+
+def await_transcription_finish(id: str, api_key: str) -> bool:
+    count = 0
+    while count < (TRANSCRIPT_WAIT_MAX_TIME/TRANSCRIPT_ITERATION_WAIT_TIME):
         # Sleep for 10
         # This MAY be a bit long but keeps possible impact on performance as low as possible
-        time.sleep(10)
+        time.sleep(TRANSCRIPT_ITERATION_WAIT_TIME)
         r = requests.get(
             "http://localhost:8393/transcriptions",
```

Comment:
not part of this change, but shouldn't this be from config file?

Reply:
Could be, yes...but if I made everything within the programm configurable it would honestly be more likely to be unusable and confusing rather than helpful